islutils/builders.h

```diff
@@ -89,6 +89,30 @@ ScheduleNodeBuilder set(std::vector<ScheduleNodeBuilder> &&children);
 
 ScheduleNodeBuilder subtree(isl::schedule_node node);
 
+template <typename T>
+std::vector<ScheduleNodeBuilder> sequenceTransform(T t) {
+  return {t};
+}
+
+template<typename T, class ...Args>
+std::vector<ScheduleNodeBuilder> sequenceTransform(T t, Args... args)
+{
+   std::vector<ScheduleNodeBuilder> vv = sequenceTransform(args...);
+   std::vector<ScheduleNodeBuilder> vv2 = {t};
+   vv2.insert(std::end(vv2), std::begin(vv), std::end(vv));;
+   return vv2;
+}
+
+template<class ...Args>
+ScheduleNodeBuilder sequence(Args... args) {
```

Comment:
Why this wouldn't clash with the similar definition in line 68 if all the arguments are `ScheduleNodeBuilder` ?  Maybe add a test for this, just to see it compiles.